spec/tictactoe_spec.rb

```diff
@@ -1,9 +1,10 @@
 require "spec_helper"
 require "tictactoe"
 
-RSpec.describe "TicTacToe" do
-  class FakeDisplay
+RSpec.describe TicTacToe do
+  class TicTacToe::FakeDisplay
```

Comment:
It's been some time since I've worked in Ruby. Did you need to namespace this `FakeDisplay` due to it already being defined in the `player_spec`? Would there be any value in moving `FakeDisplay` out into its own helper class instead of defining it twice?

Reply:
Exactly.  We debated on whether to move or mess with the name, ultimately we chose this path because the Fakes were slightly different (one uses the input method the other doesn't).  I think worth pulling out into a helper class as we rely on it more.